spec/services/lambda_jobs/git_ref_spec.rb

```diff
@@ -1,5 +0,0 @@
-RSpec.describe LambdaJobs::GIT_REF do
-  it 'exists' do
-    expect(LambdaJobs::GIT_REF).to be
```

Comment:
WDYT of removing the constant itself from the `Gemfile` and just have the gem be a normal tagged version dependency?

Reply:
yep, makes sense